src/modules/FlipImage/Module.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Display only one color channel
+ */
+module.exports = function Channel(options, UI) {
```

Comment:
Great Work! Could you please change the comments to something more appropriate and also changing the function name would be nice!

Reply:
Oh I had copied the boilerplate from channel. will change that soon 😆😆

Reply:
Changes Done!

Reply:
Will add the dist later.

Reply:
Please review the current changes